client/src/app/site/agenda/components/agenda-list/agenda-list.component.ts

```diff
@@ -77,4 +78,37 @@ export class AgendaListComponent extends ListViewBaseComponent<ViewItem> impleme
     public onPlusButton(): void {
         this.router.navigate(['topics/new'], { relativeTo: this.route });
     }
+
+    /**
+     * Handler for deleting multiple entries. Needs items in selectedRows, which
+     * is only filled with any data in multiSelect mode
+     */
+    public deleteSelected(): void {
```

Comment:
during testing this had not effect

Reply:
The agenda repository service still has a lot of dummy functions; deleting is one of them.
Should the agenda repository be part of this PR, too? Then I'll try to implement deletion, updating, etc. there, too.

Reply:
In this case you can leave it out.

I "suppose" the agenda Repo would need to resolve which repository the content object would belong (somehow).
Since I am not sure about the desired behavior, dummy functions are fine in this case for now